lib/features/modeling/behavior/RemoveParticipantBehavior.js

```diff
@@ -41,8 +49,70 @@ export default function RemoveParticipantBehavior(eventBus, modeling) {
     }
   }, true);
 
+  this.postExecute('shape.replace', function(context) {
+    var oldShape = context.oldShape,
+        newShape = context.newShape;
+
+    if (!isCollapse(oldShape, newShape)) {
```

Comment:
Better rename this to `isParticipantCollapse`?

Reply:
👍🏻